packages/block-library/src/post-featured-image/index.php

```diff
@@ -20,7 +20,8 @@ function render_block_core_post_featured_image( $attributes, $content, $block )
 	$post_ID = $block->context['postId'];
 
 	$size_slug      = isset( $attributes['sizeSlug'] ) ? $attributes['sizeSlug'] : 'post-thumbnail';
-	$featured_image = get_the_post_thumbnail( $post_ID, $size_slug );
+	$post_title     = get_the_title( $post_ID );
+	$featured_image = get_the_post_thumbnail( $post_ID, $size_slug, array( 'alt' => $post_title ) );
```

Comment:
We probably should `strip_tags` from the post title. The `wp_get_attachment_image` does this internally.

Reply:
Good point! Done 👍